notebooks/VAD

```diff
@@ -0,0 +1,37 @@
+from scipy import signal
```

Comment:
I would strongly prefer `import scipy.signal`

Reply:
which means that the callsite (`signal.medfilt`) should become `scipy.signal.medfilt`